server/models/Review.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const ReviewSchema = new mongoose.Schema({
+  reviewerId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  profileId: {
```

Comment:
```suggestion
  revieweeId: {
```
Calling it 'profileId' seems misleading as we're storing a ref to the User model here

Reply:
changed to suggested.